src/NumberInput.css

```diff
@@ -1,3 +1,30 @@
 .NumbInputField {
+  align-self: center;
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+
   margin-top: 20px;
+  padding: 0.5em 1.5em;
+
+  width: 95vw;
+  max-width: 360px;
+  height: 100px;
+
+  background: linear-gradient(300.63deg, #412027 45.56%, #562736 85.31%);
```

Comment:
The gradient is a nice touch but it is very subtle. Styling is always subjective, but maybe make it a little bit more obvious?

Reply:
yes i agree with that too, adriana! very good point

Reply:
Oh no, I guess I have to talk to my designer again XD
Thanks you two for your input on that!